composer.json

```diff
@@ -53,7 +53,7 @@
         "laravel/framework": "^9.2",
         "livewire/livewire": "^2.7",
         "lychee-org/nestedset": "^6",
-        "lychee-org/php-exif": "^0.7.13",
+        "lychee-org/php-exif": "^0.7.14",
```

Comment:
Just our of curiosity: Why do we always update the `composer.json`, too, even if we only increase patch levels?

IMHO, this is an unnecessary step. `composer update` creates a `composer.lock` which resolves all dependencies to its most recent version. IMHO, `composer.json` only defines a lower limit. This means `lychee-org/php-exif": "^0.7.13` also includes 0.7.14 if this version is published.

Reply:
True, but why not? It's a minimal step (compared to opening and reviewing a PR, at least) and may catch some edge cases.

Reply:
I didn't realize that `^0.7.13` includes 0.7.14 (I've never actually bothered to find out what this syntax means -- where is it documented?). That's why I keep complaining about it, because I assume that @ildyria simply keeps forgetting to commit `composer.json` :smiley:.

Reply:
https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Reply:
One of the reason why would be that we enforce the bug fix of #1548 rather than depending on the `.lock` especially if someone decides to stupidly do `composer update --prefer-lowest`.